recipe/meta.yaml

```diff
@@ -10,86 +10,19 @@ package:
 
 source:
   fn: cfe-{{ version }}.src.tar.xz
-  url: http://llvm.org/releases/{{ version }}/cfe-{{ version }}.src.tar.xz
+  url: https://github.com/albertziegenhagel/clang/archive/ee1e5b9d368525de4bcdebc8cb470bdc4e70aec4.tar.gz
```

Comment:
@xoviat, this branch doesn't have all the changes in https://github.com/isuruf/clang/tree/windows

Reply:
You're working off of 4.0. This branch is working off 5.0.

Reply:
I know. Then the patches there needs to be backported to 5.0 branches. Otherwise this package will not be able to compile other software like OpenBLAS and will be defining wrong macros like `__linux__`

Reply:
The diff is not that much to rebase on 5.0: https://github.com/flang-compiler/clang/compare/flang_release_40...isuruf:windows

I can do it if you add me..

Reply:
I was trying to communicate earlier that it was not a good idea to mess with 4.0, but failed. That's on me.